libraries/cli/include/cli/default_config.hpp

```diff
@@ -189,6 +189,10 @@ const char *default_json = R"foo({
         }
       }
     },
+    "gas_price" : {
+      "blocks" : "0xc8",
+      "percentile" : "0x3c"
```

Comment:
I think using integer will be better

Reply:
there is separate task for that https://github.com/Taraxa-project/taraxa-node/issues/1473, so I would keep as it is for now (to have it uniform) and we should change multiple values to make it human readable